app/controllers/publish/providers/allocations_controller.rb

```diff
@@ -9,8 +9,97 @@ def index
         )
       end
 
+      def new_repeat_request
+        authorize(Allocation)
+        provider
```

Comment:
is it worth cleaning up the instance_var (eg, `provider, `training_provider` loading into before_actions for the relevant action? Or did you want to keep it explicit when they're used/loaded

Reply:
https://github.com/DFE-Digital/teacher-training-api/blob/1a0a404b9772902f9460d26ffbd2642f4ee04b98/docs/adr/0004-before-action-variables.md